proximo-server/kafka.go

```diff
@@ -143,3 +145,34 @@ func (h *kafkaHandler) HandleProduce(ctx context.Context, topic string, forClien
 		}
 	}
 }
+
+func (h *kafkaHandler) Status() (bool, []error) {
+	errs := make(chan error)
```

Comment:
Is it worth making this a buffered channel (number of brokers) such that we're not blocked when reading

Reply:
@ggordan If I be honest, I copied this method from other ops project. Didn't want to do any magic about it here.